.woodpecker.star

```diff
@@ -337,14 +337,18 @@ config = {
         "architectures": ["arm64", "amd64"],
         "production": {
             # NOTE: need to be updated if new production releases are determined
-            "tags": ["2.0", "3.0"],
+            "tags": ["2.0"],
             "repo": docker_repo_slug,
             "build_type": "production",
         },
         "rolling": {
             "repo": docker_repo_slug + "-rolling",
             "build_type": "rolling",
         },
+        "daily": {
```

Comment:
I do not understand this change. We are not building daily on the stable branch.

Reply:
no, we don't have daily in main. and It doesn't work in stable because` build and push` works only when we have tag: https://github.com/opencloud-eu/opencloud/pull/845/files#diff-0d2cb487e7e8bfe77cea5a2347a1987e5c8864b49ffdc1a75543dba849928a74R1645-R1648

> We should be able to use the same woodpecker config for main and stable branches.
but you suggested to have similar woodpecker config for main and stable branches. That's why I left all how in main

I would also delete `rolling` and `daily` from config for stable to don't confuse

Reply:
okay. Let us prefer similarity of the files.